### ISSUE TITLE
Remove RasterValue::NoData and just return an Option<RasterValue>

### ISSUE THREAD

**Issue by mfbehrens** [pka/georaster#5]
If there is not value preset I would expect a None variant and not a no data data variant

`geotiff.rs:410`: can just be removed since the None Option is specifically mapped to the NoData variant

`geotiff.rs193`: `pub fn read_pixel(&mut self, x: u32, y: u32) -> RasterValue {` becomes `pub fn read_pixel(&mut self, x: u32, y: u32) -> Option<RasterValue> {`

**Comment by pka** [comment]
You're right, `None` is more idiomatic Rust than `NoData`. But I'm hesitating to change that, because of the long history of `NoData` in raster processing.

**Comment by pka** [comment]
With the latest `TryInto` impls `NoData` values can be converted to `None` using `ok()`:

```
assert_eq!(tiff.read_pixel(128, 64), RasterValue::NoData);
assert_eq!(f64::try_from(tiff.read_pixel(128, 64)).ok(), None);
```
